corehq/apps/cloudcare/templates/formplayer/menu_list.html

```diff
@@ -109,7 +109,7 @@ <h3><%- displayText ? displayText : "" %></h3>
     <div class="page-header menu-header">
       <h1 class="page-title"><%- title %></h1>
     </div>
-    <div class="<% if (environment === hqImport("cloudcare/js/formplayer/constants").PREVIEW_APP_ENVIRONMENT) { %>container<% } %>">
+    <div class="<% if (isAppPreview) { %>container<% } %>">
```

Comment:
Would you say in general that it is best practice to rely on variable instead of function calls/imports inside the templates?

Reply:
yes, definitely